src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -13,141 +13,65 @@
 (defn- create-section [& children]
   [:div {:style {:padding "1em 0 2em 0"}} children])
 
-(defn- reload-attributes [props state on-success]
-  (endpoints/call-ajax-orch
-    {:endpoint (endpoints/get-workspace (:workspace-id props))
-     :on-done  (fn [{:keys [success? get-parsed-response status-text]}]
-                 (if success?
-                   (let [response (get-parsed-response)
-                         ws (response "workspace")
-                         attrs-list (mapv (fn [[k v]] [k v]) (ws "attributes"))]
-                     (swap! state assoc :server-response
-                       {:attrs-list attrs-list} :loaded-attrs? true :saving? true)
-                     (on-success))
-                   (swap! state assoc :server-response {:error-message status-text})))}))
+(defn view-attributes [state refs]
+  [:div {:style {:margin "45px 25px"}}
```

Comment:
This is causing the attributes section to be indented over weirdly.  There are a lot of nested divs here--are they all necessary?